geomstats/geometry/euclidean.py

```diff
@@ -118,7 +119,7 @@ class EuclideanMetric(RiemannianMetric):
 
     def __init__(self, dim, default_point_type='vector'):
         super(EuclideanMetric, self).__init__(
-            dim=dim, signature=(dim, 0, 0),
+            dim=dim, signature=(dim, 0),
```

Comment:
Don't we use signatures with three elements: number of positive, negative, and 0 eigenvalues of the metric matrix? Why this change?

Reply:
We actually never use 0 eingenvalues, and I think we should only support non degenerate bilinear forms

Reply:
By the way, I think Minkowski should be renamed Lorentz because Minkowski seems specific to 4d ([wikipedia](https://en.wikipedia.org/wiki/Minkowski_space#Minkowski_metric))